Simplenote/Classes/InterlinkViewController.swift

```diff
@@ -52,12 +47,26 @@ class InterlinkViewController: UIViewController {
 //
 extension InterlinkViewController {
 
-    /// Relocates the receiver so that it shows up around a given Keyword in a **Sibling TextView**
-    /// - Important: We'll start listening for Content Offset changes, and the UI will be automatically repositioned
+    /// Relocates the receiver so that it shows up **around** the specified Anchor Frame, in a given ViewPort.
+    /// - Important: Both frames must be expressed in Window Coordinates. Capisce?
+    ///
+    func relocateInterface(around anchor: CGRect, in viewport: CGRect) {
+        let anchorFrame = view.convertFromWindowCoordinates(anchor)
+        let editingRect = view.convertFromWindowCoordinates(viewport)
```

Comment:
I'm not sure if we need a category for it, cause it's just
```
let anchorFrame = view.convert(anchor, from: nil)
let editingRect = view.convert(viewport, from: nil)
```
Specifying `nil` will convert given rect from a window coordinate system

Reply:
I forgot about that one!!! thank youuu!!